lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -64,6 +64,9 @@ function dosomething_global_form_campaign_node_form_alter(&$form, &$form_state,
       }
     }
     $form['revision_information']['#access'] = FALSE;
+
+    $hide_tabs_style = 'ul.tabs.secondary { display: none }';
+    drupal_add_css($hide_tabs_style, $option['type'] = inline);
```

Comment:
meh, this isn't really best practice. this should be available in `hook_menu_local_tasks_alter()`
if you're having trouble seeing it there, you might need to play with the module weight